fbw-a380x/src/systems/instruments/src/MFD/pages/SURV/MfdSurvControls.tsx

```diff
@@ -100,6 +100,16 @@ export class MfdSurvControls extends DisplayComponent<MfdSurvControlsProps> {
     this.xpdrState.sub(() => this.xpdrStatusChanged(), true);
     this.xpdrSetAltReportingRequest.sub(() => this.xpdrStatusChanged(), true);
 
+    sub
+      .on('tcas_alert_level')
+      .whenChanged()
+      .handle((val) => this.tcasTaraSelectedIndex.set(val));
+
+    sub
+      .on('tcas_direction')
+      .whenChanged()
```

Comment:
Best: Use ConsumerSubject.
Better: No need for whenChanged, as the Subject equality function checks that anyway.